charts/sourcegraph/templates/searcher/searcher.StatefulSet.yaml

```diff
@@ -59,11 +57,7 @@ spec:
           {{- $item | toYaml | nindent 10 }}
         {{- end }}
         - name: SEARCHER_CACHE_SIZE_MB
-          valueFrom:
-            resourceFieldRef:
-              containerName: searcher
-              resource: requests.ephemeral-storage
-              divisor: 1M
+          value: {{ trimSuffix "G" .Values.searcher.storageSize | mul 500 | quote }}
```

Comment:
@abeatrix accounted for the cache size env variable now.

```
❯ k exec -it symbols-0 -- /bin/sh
/ $ env| grep "SYMBOLS_CACHE_SIZE_MB"
SYMBOLS_CACHE_SIZE_MB=6000
/ $ exit
```

set properly in the pod